events/nats_subscribe.go

```diff
@@ -14,7 +14,7 @@ func (c *NATSConnection) coreSubscribe(ctx context.Context, subject string) (<-c
 		"nats.subject", subject,
 	)
 
-	sub, err := c.conn.SubscribeSync(subject)
+	sub, err := c.conn.QueueSubscribeSync(subject, c.cfg.QueueGroup)
```

Comment:
Should we be using the `NATSConsumerDurableName` function for this? https://github.com/infratographer/x/blob/f294a5bf54131ccafffbb24190d1d4a301ad571e/events/nats_connection.go#L146

Reply:
Probably, actually - though maybe we should also rename that function if we do.

Reply:
Done!